components/map/basic/src/leaflet/shapes/Polygons.js

```diff
@@ -66,16 +69,27 @@ export default class SearchMapPolygons {
   }
 
   setPolygonsOnMap({map, polygons}) {
+    let bounds
     this.removePolygonsFromMap(map)
 
     if (!(polygons instanceof Array)) {
       polygons = [polygons]
     }
 
     polygons.map(polygon => {
```

Comment:
Como sugerencia... Sé que no viene de esta pr pero si dentro del loop estamos generando side effects `this.printPolygonOnMap({fitBound, map, polygon})` y el loop no retorna nada deberíamos usar `forEach` en vez de `map`.